packages/packages.csv

```diff
@@ -77,22 +77,20 @@ Microsoft.AspNetCore.Rewrite,ship,include
 Microsoft.AspNetCore.Routing,ship,include
 Microsoft.AspNetCore.Routing.Abstractions,ship,include
 Microsoft.AspNetCore.Routing.DecisionTree.Sources,noship,exclude
+Microsoft.AspNetCore.Sockets,ship,include
 Microsoft.AspNetCore.Server.IISIntegration,ship,include
 Microsoft.AspNetCore.Server.IntegrationTesting,noship,exclude
 Microsoft.AspNetCore.Server.Kestrel,ship,include
 Microsoft.AspNetCore.Server.Kestrel.Https,ship,include
 Microsoft.AspNetCore.Server.WebListener,ship,include
 Microsoft.AspNetCore.Session,ship,include
-Microsoft.AspNetCore.SignalR.Messaging,noship,exclude
-Microsoft.AspNetCore.SignalR.Redis,noship,exclude
-Microsoft.AspNetCore.SignalR.Server,noship,exclude
-Microsoft.AspNetCore.SignalR.ServiceBus,noship,exclude
-Microsoft.AspNetCore.SignalR.Sources,noship,exclude
-Microsoft.AspNetCore.SignalR.SqlServer,noship,exclude
+Microsoft.AspNetCore.SignalR.Redis,ship,include
+Microsoft.AspNetCore.SignalR,ship,include
 Microsoft.AspNetCore.StaticFiles,ship,include
 Microsoft.AspNetCore.TestHost,ship,exclude
 Microsoft.AspNetCore.Testing,noship,exclude
 Microsoft.AspNetCore.WebSockets,ship,include
+Microsoft.AspNetCore.WebSockets.Internal,noship,exclude
```

Comment:
I thought we were planning on renaming these first: https://github.com/aspnet/SignalR/issues/24